bin/run-pipeline-yarn.sh

```diff
@@ -43,21 +43,21 @@ echo "Using SPARK_SUBMIT=$SPARK_SUBMIT"
 
 echo "RUNNING ON THE CLUSTER" 
 # TODO: Figure out a way to pass in either a conf file / flags to spark-submit
-KEYSTONE_MEM=${KEYSTONE_MEM:-1g}
-KEYSTONE_MEM=120g
-export KEYSTONE_MEM
 
 export LD_LIBRARY_PATH=/home/eecs/vaishaal/gcc-build/lib64:/home/eecs/vaishaal/gcc-build/lib:/home/eecs/vaishaal/openblas-install/lib
 export CPATH=/home/eecs/vaishaal/gcc-build/include
 
+echo CORES $SPARK_EXECUTOR_CORES
+echo NUM EXECUTORS $SPARK_NUM_EXECUTORS
+
 # Set some commonly used config flags on the cluster
 "$SPARK_SUBMIT" \
   --master $MASTER \
   --class $CLASS \
-  --num-executors 32 \
+  --num-executors  $SPARK_NUM_EXECUTORS \
```

Comment:
we aren't going to really be using the bin/scripts much anyway, but yeah we can.